azure/apt.yml

```diff
@@ -3,6 +3,18 @@ parameters:
 
 steps:
   - script: |
+      if ! grep -r -E '^deb.*xenial-backports.*' /etc/apt/sources.list* > /dev/null; then
+      cat <<EOT | sudo tee /etc/apt/sources.list.d/xenial-backports.list > /dev/null
+      deb http://archive.ubuntu.com/ubuntu/ xenial-backports main restricted universe multiverse
+      EOT
+      fi
+
+      cat <<EOT | sudo tee /etc/apt/preferences.d/xenial-backports-lmdb > /dev/null
+      Package: liblmdb-dev liblmdb0 lmdb-doc
+      Pin: release a=xenial-backports
+      Pin-Priority: 500
+      EOT
```

Comment:
Why is this needed?

Reply:
The lmdb package in the regular repo has no pkg-config definitions.

Reply:
Hrm. That's unfortunate, but means that we should not use pkg-config for lmdb at this time after all :( Or at least not exclusively. Ubuntu 16.04 is our main baseline for what should be working out of the box.

Reply:
Ok, that’s good to know. I’ll look into this further.

Reply:
How does this compare with argon2 needing a third party repo to build?

Reply:
The difference is just that here it used to work, but won't (without some hoops) after this change, while with argon2 it never worked in the first place :) When adding optional dependencies to very new libraries, it is unavoidable that they won't be available on older distros.

For argon2 in particular, the issues surrounding builds with libargon2 motivated the addition of libsodium as an additional argon2 backend in 7.4, which is both more widely available and easier to deal with for packagers.